### ISSUE TITLE
[Bug]: MessageBar does not reflow correctly when zoomed to 400%

### ISSUE THREAD

**Issue by YoelEigner** [microsoft/fluentui#29057]
### Library

React / v8 (@fluentui/react)

### System Info

```shell
N/A
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://developer.microsoft.com/en-us/fluentui#/controls/web/messagebar

## Actual Behavior
The Success/Warning `MessageBar` component reflows incorrectly when zoomed in to 400% if the screen size is larger the 479px, the text being compressed and is not accessible to all users.

## Expected Behavior
The Success/Warning `MessageBar` component should reflow correctly when zoomed in to 400%.

This bug mentioned in #27749 was not fixed for screen sizes over 479px

![image](https://github.com/microsoft/fluentui/assets/35924447/cb1af877-13f1-4cef-8cd4-517fcc162051)

## Reproduction
This issue can be reproduced on the docsite's MessageBar page when setting the screen size to > 479px

### Requested priority

Normal

### Products/sites affected

MS Defender

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by TristanWatanabe** [comment]
@YoelEigner I'm unable to repro this on the latest Edge and Chrome browsers. Is this still an issue?

<img width="1106" alt="image" src="https://github.com/microsoft/fluentui/assets/8649804/b499d873-9f88-489b-b757-54b1db098197">

<img width="1112" alt="image" src="https://github.com/microsoft/fluentui/assets/8649804/e7508639-1a54-4734-bcf7-0b4f079950ba">

**Comment by YoelEigner** [comment]
@TristanWatanabe Yes, if the screen is greater then 479px this issue remains. 

You can also see in the fix [here](https://github.com/microsoft/fluentui/pull/28100) that this won't work above 479px

**Comment by TristanWatanabe** [comment]
Finally able to repro after turning on browser emulation and setting

**Comment by YoelEigner** [comment]
I've noticed this there is also an issue on all screen sizes one zoom less then 400% 
200% and 300% will still have this issue
<img width="420" alt="image" src="https://github.com/microsoft/fluentui/assets/35924447/3ea8719d-3ccf-4daa-9547-d499c8602a16">
